### ISSUE TITLE
network weighting 

### ISSUE THREAD

**Issue by stevefoy** [MarvinTeichmann/MultiNet#38]
Very good work Marvin, 

If possible could you please explain,  the training weighting for training logic?? In the paper its hard to follow what exactly you did.

I see your weights in the .json files and the subgraph training selection is based on this logic train.py.  The weight values are now [1, 0] seems to indicate you only uses the subgraph[0] , so the segmentation graph only. The training results show the detection subgraph working, As i see a car being detected in the bounding box. While the loss function seems to be using both graphs regardless of these weights.

Could you please give some insight, your weighting technique in the following ? 

**File: train.py
lt is alway 0 if using multinet2.json so this is fully weighted on the  "segmentation" graph then?  
**line 202:** `weights = meta_hypes['selection']['weights']`
**line 229:** `sess.run([subgraph[model]['train_op']], feed_dict=feed_dict)`

**File:multinet2.json** 
 **"weights": [1, 0]** and older commits had **"weights": [1, 2]**
```
 "model_list": ["segmentation", "detection"],
    "selection": {
        "random": false,
        "use_weights": true,
        "weights": [1, 0]
    }
```,
